data/projects/coala_cli.md

```diff
@@ -19,32 +19,84 @@ project is going to rely heavily upon mockups and feedback from the
 community. Also, in addition to the above changes the applicant is encouraged
 to come up with their own ideas to improve the CLI.
 
```

Comment:
but if we use checkboxes, no more need for italic. can you please remove the italic at all from the 2nd commit and remove it here too therefore and just keep the checkboxes? or just delete this commit and make this change with checkbxoes instead of italic in the 2nd commit? thatd be all and its good to merge, thanks :)